src/uct/ib/rdmacm/rdmacm_cm_ep.h

```diff
@@ -41,10 +42,14 @@ typedef struct uct_rdmacm_cm_ep {
 enum {
     UCT_RDMACM_CM_EP_ON_CLIENT     = UCS_BIT(0),
     UCT_RDMACM_CM_EP_ON_SERVER     = UCS_BIT(1),
-    UCT_RDMACM_CM_EP_CONNECTED     = UCS_BIT(2),
-    UCT_RDMACM_CM_EP_DISCONNECTING = UCS_BIT(3) /* uct_ep_disconnect was called
-                                                   on the ep. this ep is not
-                                                   necessarily disconnected yet */
+    UCT_RDMACM_CM_EP_CONNECTED     = UCS_BIT(2), /* connection was successfully
```

Comment:
didn't we say that this flag would be renamed to something like connect_cb_called?
same for the disconnect.

Reply:
No, I would keep the state based on lower RDMACM events instead of user level. What if user wants to disconnect from connect_cb?

Reply:
why is this a problem if the user wants to disconnect from the connect_cb? we can still use the flags.

Reply:
but if it's called "connect_cb_called" this is mean it should be set "after" cb call, right? then the flag won't be set inside the callback and disconnect will return busy

Reply:
not necessarily since you are calling the connect_cb right after but also can call it something like got_connect_event. there isn't a real connect event in rdmacm since on both sides they are different - established on the server and response on the client, but we can "unite" them to a "connect_event" like we have a disconnect_event.

Reply:
renamed